rllib/offline/estimators/direct_method.py

```diff
@@ -62,50 +63,38 @@ def __init__(
         ), "self.model must implement `estimate_v`!"
 
     @override(OffPolicyEstimator)
-    def estimate(self, batch: SampleBatchType) -> Dict[str, Any]:
-        """Compute off-policy estimates.
+    def estimate_multi_step(self, episode: SampleBatch) -> Dict[str, float]:
```

Comment:
I am now separating episodic RL logic from bandits which will be easier to maintain and debug. It will also focus on our current use-case which is bandits.